src/script/util/ReactionUtil.ts

```diff
@@ -43,3 +43,9 @@ export function groupByReactionUsers(reactions: Reactions): ReactionsGroupedByUs
 export function getEmojiUrl(unicode: string) {
   return `/image/emojis/img-apple-64/${unicode}.png`;
 }
+
+export function sortReactionsByUserCount(reactionsList: [string, string[]][]) {
+  return reactionsList.sort(
+    ([, reactionAUserList], [, reactionBUserList]) => reactionBUserList.length - reactionAUserList.length,
```

Comment:
maybe some comments are good to explain that the reaction list is an array and the first value is the name of the reaction and not used for sorting? Could be helpful for people in the future possibly.